package-lock.json

```diff
@@ -1,172 +1,6375 @@
 {
   "name": "openpgp",
   "version": "5.0.0-3",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Since package.json is not changed in this PR, please don't change package-lock.json, either